kolibri/plugins/facility/assets/src/routes.js

```diff
@@ -25,20 +26,28 @@ import {
 import { PageNames } from './constants';
 
 function facilityParamRequiredGuard(toRoute, subtopicName) {
+  const { isNavigationFailure, NavigationFailureType } = VueRouter;
   if (store.getters.userIsMultiFacilityAdmin && !toRoute.params.facility_id) {
-    router.replace({
-      name: 'ALL_FACILITIES_PAGE',
-      params: { subtopicName },
```

Comment:
reintroducing this line here, `params: { subtopicName }`, should resolve the problem mentioned in my review comment